utils/db.ts

```diff
@@ -481,21 +477,27 @@ export async function deleteUserBySession(sessionId: string) {
 
 /** @todo Migrate to ["users", login] key */
 export async function getUser(login: string) {
-  return await getValue<User>(["users", login]);
+  const usersByLoginKey = ["users", login];
+  const res = await kv.get<User>(usersByLoginKey);
+  return res.value;
 }
 
 export async function getUserBySession(sessionId: string) {
```

Comment:
Previously, this function performed 1 eventual read and _possibly_ 1 consistent read. Now, with your changes, it performs both every time. Let's fix that up.